compiler/rustc_const_eval/src/const_eval/mod.rs

```diff
@@ -131,38 +129,3 @@ pub(crate) fn try_destructure_mir_constant<'tcx>(
 
     Ok(mir::DestructuredConstant { variant, fields })
 }
-
-#[instrument(skip(tcx), level = "debug")]
-pub(crate) fn deref_mir_constant<'tcx>(
```

Comment:
So happy to see this go. :)

Now, when will destructure_mir_constant disappear? ;)
It seems to be used only for pretty-printing, and for some obscure SIMD-related thing.

Reply:
yea, the SIMD thing I wanna solve first, and then just move `destructure_mir_constant` into pretty printing and kill the query